.gitlab-ci.yml

```diff
@@ -1,126 +1,21 @@
 stages:
-    - empty
+    - build-test
 
-dummy:
-  stage: empty
+trigger-simnet-for-build-and-test:
+  stage:                     build-test
+  variables: 
+    # The branch or tag name for which project is built. 
+    TRGR_REF:                "${CI_COMMIT_REF_NAME}"
+  image:                     "docker.io/paritytech/tools:latest"
   script:
-      - echo nothing to see here
+    - echo 'TRGR_REF is ${TRGR_REF}'
+    - ./scripts/trigger_pipeline.sh
+        --simnet-version=master
+        --upstream-project=simnet_scripts
+        --upstream-ref="${TRGR_REF}"
+        --downstream-id=332
+        --image-name=""
+        --image-tag=""
```

Comment:
what do these empty values mean? Maybe better explicitly specify the stable or master image of substrate/polkadot ?

Reply:
then why mentioning flags at all? In substrate CI you use just one flag

Reply:
there is the bash safety option
`set -eou pipefail`  I think  the -u  that check if a variable is unset

Reply:
I could give it default value like
https://github.com/paritytech/simnet_scripts/blob/afea78c2bab793b1ead4b39678ffe2f161bfb0ff/scripts/trigger_pipeline.sh#L11
`SIMNET_VERSION=""`  if you prefer this option

Reply:
I mean https://github.com/paritytech/substrate/blob/c3d30ac11f110062ab2033f2b3ebad51db90bd37/.gitlab-ci.yml#L714 there you just pass it one flag having everything else optional. Or is it connected with that here `trigger_pipeline.sh` is different?

Reply:
https://github.com/paritytech/substrate/blob/c3d30ac11f110062ab2033f2b3ebad51db90bd37/.gitlab-ci.yml#L714
here is just 1 arg because you set the rest of variables in the CI before calling the script